src/system.jl

```diff
@@ -2,6 +2,9 @@ const MARKET_WIDE_ZONE = -9999
 const BidName = InlineString31
 const ZoneNum = Int64
 
+
+abstract type SystemRequirements end
```

Comment:
Since the requirements and ancillary services are linked across multiple fields, would it be better to develop the new market as a separate subtype of System to avoid major breaking changes to the package?

Reply:
I'm debating the pros and cons of that.  Having a separate System subtype would keep us free from major breaking changes while developing the second market.  On the other hand it's probably good for us to think about common parts of the API from the start, because it'll save us time and energy later when we come to reconcile the two.